observations/obs_converters/even_sphere/README.rst

```diff
@@ -8,69 +8,113 @@ generate a vertical column of observations.  This could mimic
 a radiosonde observing network, for example.
 
 
-
-This directory contains a MATLAB script that generates
-input for the 'create_obs_sequence' program.  It takes
-a number of vertical levels and a total number of points,
-and generates a roughly evenly distributed set of observations
+This directory contains MATLAB scripts (even_sphere*.m) 
```

Comment:
the asterisk does something funny to the formatting.  maybe you don't have to specify the name since anything that's .m is a matlab script?

```suggestion
This directory contains MATLAB scripts 
```